### ISSUE TITLE
language strings.json key conflict

### ISSUE THREAD

**Issue by sh0wer1ee** [cecilbowen/pokemon-sandwich-simulator#20]
![fig1](https://user-images.githubusercontent.com/18623532/208245675-dccdc23c-94d2-4152-be60-a653e8054cb4.png)
As shown in this figure, material `Egg` and filter `Egg` (Egg Power) share the same string key in `language/strings.json`.
However, in other localiztions these two strings may not be identical (In fact the differences are huge).
For example:
| language name   | egg (item)  | egg (abbr. of egg power) |
|  ----  | ----  | ----  |
| simplified Chinese  | 水煮蛋片 | 蛋蛋（蛋蛋力） |
| traditional Chinese | 水煮蛋片 | 蛋蛋（蛋蛋力） |

Add new string key (line#11 & #97) and adjust label in code can solve this issue.
```diff
+  "Egg(item)": "Egg",
-   "Egg": "Egg",
...
+ "Egg(power)": "Egg",
    "Catch": "Catching",
```

**Comment by cecilbowen** [comment]
I noticed this when putting together the language files, it's just that I hadn't run into any languages where it was a huge difference yet.  I will update the files, though, since now there is a language where it is a big difference.  We can call the egg ingredient "Sliced Egg" instead.

**Comment by cecilbowen** [comment]
Fixed with [05a9461](https://github.com/cecilbowen/pokemon-sandwich-simulator/commit/05a9461b02fd06e29c35cdea7e1dc772e5440e2e)
